TLM/TLM/Manager/Impl/TrafficPriorityManager.cs

```diff
@@ -72,8 +72,8 @@ public class TrafficPriorityManager
             ref NetNode netNode = ref Singleton<NetManager>.instance.m_nodes.m_buffer[nodeId];
 
             if (!netNode.m_flags.CheckFlags(
-                    NetNode.Flags.Created | NetNode.Flags.Deleted | NetNode.Flags.Junction,
-                    NetNode.Flags.Created | NetNode.Flags.Junction)) {
+                    NetNode.Flags.Created | NetNode.Flags.Junction,
+                    NetNode.Flags.Created | NetNode.Flags.Deleted | NetNode.Flags.Junction)) {
```

Comment:
This would work but its not right.
the forbidden flag should be only set to `NetNode.Flags.Deleted`

Reply:
I see, you are merging required with forbidden internally, I didn't notice that at first, so result is identical but more readable 👍